UVAtlasTool/Mesh.cpp

```diff
@@ -1087,6 +1089,59 @@ HRESULT Mesh::GetVertexBuffer(_Inout_ DirectX::VBWriter& writer) const
     return S_OK;
 }
 
+//--------------------------------------------------------------------------------------
+void Mesh::ExportToOBJ(const wchar_t* szFileName) const
```

Comment:
This method should return ``HRESULT`` to be consistent with the other ones.

At the end of the function:

```
os.close();
return (os.bad())  ? E_FAIL : S_OK;
```

```

Reply:
done